### PR TITLE
Add polygon tangens

### PR DESCRIPTION
**Pull request by SteffenHaeussler**
this pull request closes https://github.com/diogomatoschaves/pyturf/issues/14

Please make sure that:

- [x] You have read [CONTRIBUTING.md](../CONTRIBUTING.md)
- [x] You Include a link to the issue this pull request is associated to.
- [x] The linter has been run at the root level of the project: `black .`
- [x] All tests are passing: `python -m pytest --verbose --cov=./` 
- [x] Your branch is up to date with master: `git rebase upstream/master`

For new modules, also make sure that:

- [x] The new module function is exported in `turf/<your-module>/__init__.py` and `turf/__init__.py`.
- [x] The new module has been added under the `Available Modules` section on the [README](../README.md).

### CODE REVIEW

File: turf/polygon_tangens/tests/test_polygon_tangens.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from turf.utils.error_codes import error_code_messages
+from turf.utils.exceptions import InvalidInput
+from turf.utils.test_setup import get_fixtures
+
+current_path = os.path.dirname(os.path.realpath(__file__))
+
+fixtures = get_fixtures(current_path)
+
+
+class TestPolygonTangens:
```
Comment (diogomatoschaves) [inline turf/polygon_tangens/tests/test_polygon_tangens.py:16]: Same here: name should be `TestPolygonTangents`

File: turf/polygon_tangens/tests/test_polygon_tangens.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    @pytest.mark.parametrize(
+        "fixture",
+        [
+            pytest.param(fixture, id=fixture_name)
+            for fixture_name, fixture in fixtures.items()
+        ],
+    )
+    def test_polygon_tangens(self, fixture):
```
Comment (diogomatoschaves) [inline turf/polygon_tangens/tests/test_polygon_tangens.py:24]: Name should be `test_polygon_tangents`

File: README.md [hunk 3]
```diff
@@ -85,6 +85,7 @@ Currently, the following modules have been implemented:
 - [nearest-point](https://github.com/diogomatoschaves/pyturf/tree/master/turf/nearest_point)
 - [point-on-feature](https://github.com/diogomatoschaves/pyturf/tree/master/turf/point_on_feature)
 - [point-to-line-distance](https://github.com/diogomatoschaves/pyturf/tree/master/turf/point_to_line_distance)
+- [polygon-tangens](https://github.com/diogomatoschaves/pyturf/tree/master/turf/polygon_tangens)
```
[the remark this answers is not in the document]
Comment (SteffenHaeussler) [reply README.md:88]: omg. you are right. how embarrassing. at least, i made the same mistake the whole time :)

File: turf/polygon_tangens/_polygon_tangens.py [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    more:
+    http://geomalgorithms.com/a15-_tangents.html
+
+    :param point: point [lng, lat] or Point feature to calculate the tangent points from
+    :param polygon: polygon to get tangents from
+
+    :return:
+        Feature Collection containing the two tangent points
+    """
+    start_point = normalize_feature(start_point, ("Point"))
```
[the remark this answers is not in the document]
Comment (SteffenHaeussler) [reply turf/polygon_tangens/_polygon_tangens.py:39]: The test cases [test_input_mutation_prevention](https://github.com/SteffenHaeussler/pyturf/blob/ecfb2fedb0e69d53dd214d0ddf357f3761ba803e/turf/polygon_tangens/tests/test_polygon_tangens.py#L35) and [test_exception](https://github.com/SteffenHaeussler/pyturf/blob/ecfb2fedb0e69d53dd214d0ddf357f3761ba803e/turf/polygon_tangens/tests/test_polygon_tangens.py#L91) fail without the normalization. This function fixes the lazy typing in python. 

The normalization explicitly makes sure, that every incoming geojson is in the defining feature for the function or at least can be transformed into the defining feature. This function was indeed missing and `get_coords_from_features` doesn't cover all possible use cases of a function input. My suggestion is, that `normalize_feature`, as a high-level function, should be moved to the `invariant module`. It basically should be used in every module.

**Comment by SteffenHaeussler** [comment]
i closed this pull request, because of the naming issue. The new pull request is found [here](https://github.com/diogomatoschaves/pyturf/pull/29)

**Pull request closed without merging**